src/common/typings.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.json" {
```

Comment:
I don't believe this is needed in the version of TypeScript we use, please check and remove.

Reply:
I am not able to import without declaring the module :( Do you have any other way?

Reply:
I believe you can just use `require`. As we target CommonJS for Node first, this should work (untested):

```javascript
// tslint:disable-next-line:no-var-requires
import packageInformation = require("../../version.json");
```